azure-media-spring-boot-autoconfigure/src/main/java/com/microsoft/azure/autoconfigure/azuremedia/MediaServicesProperties.java

```diff
@@ -0,0 +1,96 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See LICENSE in the project root for
+ * license information.
+ */
+
+package com.microsoft.azure.autoconfigure.azuremedia;
+
+import javax.validation.constraints.NotNull;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+@ConfigurationProperties("azure.media")
+public class MediaServicesProperties {
+
+    @NotNull
+    private String mediaServiceUri;
+    @NotNull
+    private String oAuthUri;
+    @NotNull
+    private String clientId;
+    @NotNull
+    private String clientSecret;
+    @NotNull
+    private String scope;
+

```

Comment:
Although the SDK expects client id and client secret, I prefer the name `accountName` and `accountKey`, for that're the names shown in Azure portal.

1. Please only apply not null validation to the accountName and accountKey.
2. For the rest three variables, assign the default value if null.